File: android/app/src/main/java/io/wassa/wisexampleproject/model/DetectedObject.kt
package io.wassa.wisexampleproject.model

import com.google.gson.annotations.SerializedName

class DetectedObject {
    @SerializedName("class_name") val vehicleType: VehicleType? = null
    @SerializedName("score") val score: Double? = null
    @SerializedName("box") val box: List<Point>? = null
}

File: swift/WisImplementationExample/Models/DetectedObject.swift
//
//  DetectedObject.swift
//  WisImplementationExample
//
//  Created by Bertrand VILLAIN on 18/10/2021.
//

import Foundation

struct DetectedObject: Decodable {
  var className: VehicleType?
  var score: Double?
  var box: [Point]?
}
